packages/react-stylesheets/src/StylesheetContext.tsx

```diff
@@ -3,38 +3,44 @@ export type HTMLDirection = 'rtl' | 'ltr' | 'auto';
 
 export interface StylesheetContextType {
   target: Document | undefined;
-  registerStyles: (stylesheets: string[], context: StylesheetContextType) => void;
+  registerStyles: (stylesheets: undefined | string | string[], context: StylesheetContextType) => void;
```

Comment:
Should we maybe switch the positions of the `stylesheets` and the `context` so that we can pass context without passing undefined?

```suggestion
  registerStyles: (context: StylesheetContextType, stylesheets?: string | string[]) => void;
```

Reply:
I think it's good right now. `registerStyles` purpose is to register styles idempotently - that is, you can call it over and over with the same input and only have it render styles once. The reasoning for adding `undefined` to the typings was that in some cases, the caller may conditionally have a stylesheet based on user input (e.g. compose). Rather than requiring each usage to check for undefined, I just allow it.

But there is no scenario where calling registerStyles makes sense to pass in without stylesheets argument.